config/sidekiq_schedule.yml

```diff
@@ -0,0 +1,5 @@
+kairos_daily_updater:
+  cron: "0 3 * * *"
+  class: "DailyGroupingUpdater"
+  args:
+    app_name: "Kairos"
```

Comment:
I'll come chat with you about this since I was having trouble finding a way to schedule this without doing so.